docs/plotting.rst

```diff
@@ -0,0 +1,128 @@
+Plotting
+========
+
+Below we describe additional features available for customising spectral plots.
+
+
+Creating publication-quality plots
+----------------------------------
+
+By default, the legend will be positioned to the right of the figure box with the best-fit parameter values and the
+full model name. The automated plotting includes an alternative, more compact figure style with the legend included
+within the figure box and with an abbreviated model name and no fit info. This is done using the following code:
+
+.. code-block:: python
+
+    from pulsar_spectra.catalogue import collect_catalogue_fluxes
+    from pulsar_spectra.spectral_fit import find_best_spectral_fit
+
+    cat_dict = collect_catalogue_fluxes()
+    pulsar = 'J0034-0534'
+    freqs, fluxs, flux_errs, refs = cat_dict[pulsar]
+    best_model_name, iminuit_result, fit_info, p_best, p_category = find_best_spectral_fit(pulsar, freqs, fluxs, flux_errs, refs, plot_best=True, alternate_style=True)
+
+This will produce the following plot:
+
+.. image:: figures/example_alternate_style.png
+  :width: 800
+
+
+Using custom marker types
+-------------------------
+
+By default, the marker types will cycle through a set list of marker types. If you would like to use custom marker
```

Comment:
Just make it a bit clearer that this is useful to make multiple plots have consistent ref markers